spec/example-artifacts.json

```diff
@@ -0,0 +1,110 @@
+[
+  {
+    "system": "OFM",
+    "symbolicName": "com.oracle.ofm",
```

Comment:
Does this field match Maven's group ID?

Reply:
yes. That should actually be "name". I will update.